safe-global/web-core
# fix: `Backdrop` `backgroundColor` based on theme

## What it solves

Resolves theme-based `Backdrop` `backgroundColor`

## How this PR fixes it

The `Backdrop` `backgroundColor` is set from the theme `palette`, thus reacting to light/dark mode.

## How to test it

1. Open the Safe in light mode and open the wallet popper. Observe a light background.
2. Open the Safe in dark mode and open the wallet popper. Observe a dark background.

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/192959862-d1e4f53a-3c8e-4c04-8440-b077c1868c95.png)

![image](https://user-images.githubusercontent.com/20442784/192959836-d6945fe0-6503-444c-b67d-132741282bcc.png)
